packages/typespec-autorest/test/paging.test.ts

```diff
@@ -40,6 +40,25 @@ it("define a custom paged operation with custom next link", async () => {
   });
 });
 
+it("value is default item name for x-ms-pageable", async () => {
+  const res = await compileOpenAPI(
+    `
+    @route("t")
+    @list
+    op test(): {
+      @pageItems value: string[];
+      @nextLink next: string;
+    };
+    `,
+  );
+
+  const get = res.paths["/t"].get;
+  ok(get);
+  deepStrictEqual(get["x-ms-pageable"], {
+    nextLinkName: "next",
+  });
+});
```

Comment:
```suggestion
  deepStrictEqual(res.paths["/t"].get!.["x-ms-pageable"], {
    nextLinkName: "next",
  });
});
```
keep it a little simpler